CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ project(ipcamera_driver)
 
 
 find_package(catkin REQUIRED COMPONENTS
+  std_srvs
```

Comment:
List of packages should be sorted alphabetically.

Reply:
done